Resources/Prototypes/Catalog/VendingMachines/Inventories/discount.yml

```diff
@@ -8,3 +8,6 @@
     FoodSnackPopcorn: 3
     FoodSnackEnergy: 3
     CigPackMixed: 2
+  contrabandInventory:
+    FoodFrozenSandwich: 3
+    FoodFrozenSandwichStrawberry: 3
```

Comment:
Id rather the ice cream be a chef/kitchen thing. Makes for good RP.

Reply:
My reasoning for adding these was to give the Discount Dan's machines something actually "worth getting" that's still ultimately just junk food. Perhaps the freezer section of the machine has been locked out of regular access due to a product recall, or faults in those units, and hacking it unlocks the partially-raided selection (0-3 of each sandwich spawn in these machines).

ChefVend does make sense to include these sandwiches instead. As of now for the update, the ChefVend contains 2x Moldy Bread, 1x Egg Carton (Broken, and 1 Donk-Pocket Box with 6 Donk Pockets. I've mostly just trolled chef and given an alternate food supply for when kitchen is unstaffed. Bringing it up to 5 items feels excessive, but if I did that I'd probably cut out the Moldy Bread to make it a cleaner 4.

If we moved this, what should we include in the Discount Dan's instead?

Reply:
I have moved the ice cream sandwiches as 2-ofs to the ChefVend.